whylog/teacher/__init__.py

```diff
@@ -69,6 +69,12 @@ def _verify(self):
         """
         pass
 
+    def get_rule(self):
+        """
+        Creates rule for Front that will be shown to user
+        """
+        return create_sample_rule()
```

Comment:
Please add a `#TODO remove mock` comment so that all such code can be identified easily